optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -220,11 +221,15 @@ public void writeReport() {
         for (ProblemBenchmarkResult problemBenchmarkResult : plannerBenchmarkResult.getUnifiedProblemBenchmarkResultList()) {
             if (problemBenchmarkResult.hasAnySuccess()) {
                 for (ProblemStatistic problemStatistic : problemBenchmarkResult.getProblemStatisticList()) {
+                    for (SingleStatistic singleStatistic : problemStatistic.getSingleStatisticList()) {
+                        singleStatistic.retrievePointList();
+                    }
                     problemStatistic.writeGraphFiles(this);
                 }
                 for (SingleBenchmarkResult singleBenchmarkResult : problemBenchmarkResult.getSingleBenchmarkResultList()) {
                     if (singleBenchmarkResult.isSuccess()) {
                         for (PureSingleStatistic pureSingleStatistic : singleBenchmarkResult.getPureSingleStatisticList()) {
+                            pureSingleStatistic.retrievePointList();
                             pureSingleStatistic.writeGraphFiles(this);
```

Comment:
Should the pointList not be hibernated again after the graph is generated (or a bit later due to the problem statistic stuff)? Otherwise it remains in memory and that means that at some point all pointLists are in memory (which could still be too big for the memory even though the solver isn't running so that's not taking up memory).